components/card.css

```diff
@@ -0,0 +1,130 @@
+@import url('../styles.css');
+
+.card{
+   width: 17rem;
+   box-shadow: 3px 1px 6px 1px rgb(0 0 0 / 20%);
+   border-radius: 10px;
+   margin-left: 4rem;
+   transition: .2s all ease-in-out;
+}
+
+.card:hover{
+    box-shadow: 3px 1px 10px 1px rgb(0 0 0 / 20%);;
+}
+
+.img-content{
+    margin-top: -1rem;
+}
+
+.card img{
+    height: 100%;
+    width: 100%;
+}
+
+.card-content{
+    padding: 0.5rem;
+}
+
+.text-badge{
+    position: relative;
+    top: 2rem;
+    background-color: #E67A00;
+    width: fit-content;
+    padding: 0.2rem;
+    color: var(--white-color);
+}
+
+.item-name{
+    margin-left: 0.5rem;
+    color: var(--primary-color);
+}
+
+.item-desc{
+    margin-left: 0.5rem;
+    opacity: .5;
+}
+
+.prices{
+    display: flex;
+    gap: 7px;
+    margin-top: .5rem;
+    margin-bottom: .5rem;
+}
```

Comment:
For less than 1 rem, Vikram said we should use px.

Reply:
thanks for giving clarification, I'm really confused about that.